app/workers/license_backfill_worker.rb

```diff
@@ -11,7 +11,7 @@ def perform(platform, name)
     versions = pm.versions(json_project, project.name)
     versions.each do |vers|
       version = project.versions.find_by(number: vers[:number])
-      version.update(original_license: vers[:original_license]) if version&.original_license.nil?
+      version.update(original_license: vers[:original_license]) if version && version&.original_license.nil?
```

Comment:
This is another common error in Sidekiq right now. I assume it's when there's a new version that isn't in the db yet. Quickfix here is to move along, altho we could change line 13 to `find_or_create_by` too.